motor/discovery/aws/ssm_instances.go

```diff
@@ -202,9 +231,19 @@ func ssmInstanceToAsset(account string, region string, instance types.InstanceIn
 		}
 	}
 	// add AWS metadata labels
-	asset.Labels = addAWSMetadataLabels(asset.Labels, ssmInstanceToBasicInstanceInfo(instance, region, account))
+	asset.Labels = addAWSMetadataLabels(asset.Labels, ssmInstanceToBasicInstanceInfo(instanceInfo.instance, instanceInfo.region, instanceInfo.account))
 	if label, ok := asset.Labels[ImportedFromAWSTagKeyPrefix+AWSNameLabel]; ok {
 		asset.Name = label
 	}
 	return asset
 }
+
+func getProbableUsernameFromSSMPlatformName(name string) string {
+	if strings.HasPrefix(name, "RHEL") {
```

Comment:
Are we sure this works for all rhel? What happens if I spin up RHEL instead of CentOS?

Reply:
reel gives me ec2-user, centos gives centos. i updated to just check for centos or ubuntu, and it it's neither one, just use ec2-user